app/services/send_chase_email.rb

```diff
@@ -0,0 +1,9 @@
+class SendChaseEmail
+  def perform(reference:)
+    RefereeMailer.reference_request_chaser_email(reference.application_form, reference)
+
+    reference.application_form.application_choices.each do |application_choice|
+      application_choice.update!(status: 'awaiting_references_and_chased')
```

Comment:
```suggestion
      application_choice.send_chase_email!
```

This is a method that is auto-generated by the `event :send_chase_email` declaration in the state machine, it's better to use it instead of `update!`